dl4j-cv-labs/src/main/java/global/skymind/solution/segmentation/car/CustomLabelGenerator.java

```diff
@@ -0,0 +1,79 @@
+package global.skymind.solution.segmentation.car;
+
+import org.datavec.api.io.labels.PathLabelGenerator;
+import org.datavec.api.writable.NDArrayWritable;
+import org.datavec.api.writable.Writable;
+import org.datavec.image.loader.NativeImageLoader;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+
+public class CustomLabelGenerator implements PathLabelGenerator {
```

Comment:
refactor this class, there're quite a few of CustomLabelGenerator, kinda redundant.